deploy/k8s/charts/trustification/templates/services/bombastic/api/030-Deployment.yaml

```diff
@@ -43,8 +43,6 @@ spec:
             - "--index-mode"
             - {{ $mod.module.indexMode | default "file" | quote }}
 
-            - "--auth-configuration"
```

Comment:
Why is the full authentication configuration being removed? What about the other configuration options in there?